providers/gce/gce.go

```diff
@@ -726,6 +726,16 @@ func (g *Cloud) Region() string {
 	return g.region
 }
 
+// Regional returns true if the cluster is regional.
+func (g *Cloud) Regional() bool {
+	return g.regional
+}
+
+// Zone returns the zone.
```

Comment:
last nit to fix the godoc `LocalZone` and I'll approve

Reply:
Fixed.